pesticide_backend/src/pesticide_app/migrations/0016_auto_20220131_0020.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.1.1 on 2022-01-30 18:50
```

Comment:
A better approach is to merge all these migration files into a single file. 5 migration files for a single model are too much IMO 😅

Merge these 5 files into a single migration file.